exercises/practice/word-count/src/lib.cairo

```diff
@@ -0,0 +1,3 @@
+pub fn word_count(phrase: ByteArray) -> Felt252Dict<u64> {
+    panic!("implement `word_count`")
+}
```

Comment:
open terminal and run `./bin/format_all.sh`

Reply:
```...
Exercise word-count is just a stub, skipping
```

I get this when I run this script.

Reply:
1. Don't forget to implement [Scarb.toml](https://github.com/exercism/cairo/pull/304/files#diff-459e440a2c01aac6a634d53500311f8dc61f4202bd468951c75958eb181f8c5a).
2. Don't forget to update _lib.cairo_ to match example and tests.